routes/index/index.jade

```diff
@@ -18,7 +18,7 @@ div( class=className )
         a( href='https://goo.gl/maps/0gkOe' ) SOMA StrEat Food
         |  (in the <a href="/map" target="_self">barn</a>)
   
-  section#hero
+  section#hero: .container
```

Comment:
yeah, it's just that photos stretched beyond their intended resolution look really bad